File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.cs
public class Solution {
    public string ReversePrefix(string word, char ch) {
        int index= word.IndexOf(ch);
        string firstWord=word[0..(index+1)];
        string firstWordRev=new string(firstWord.Reverse().ToArray());
        string secondWord=word[(index+1)..];
        string result=firstWordRev+secondWord;
        return result;
    }
}

File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.js
/**
 * @param {string} word
 * @param {character} ch
 * @return {string}
 */
var reversePrefix = function(word, ch) {
    let index=word.indexOf(ch);
    let firstWord = word.slice(0, index+1).split("").reverse().join("");
    let secondWord=word.slice(index+1);
    let result =firstWord+secondWord;
    return result;
};